core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -1008,183 +1149,206 @@
   @ReplacedBy(property = INSTANCE_VOLUMES)
   INSTANCE_DFS_DIR("instance.dfs.dir", "/accumulo", PropertyType.ABSOLUTEPATH,
-      "This property is deprecated since 1.6.0. "
-          + "HDFS directory in which accumulo instance will run. "
-          + "Do not change after accumulo is initialized."),
+      "HDFS directory in which accumulo instance will run. "
+          + "Do not change after accumulo is initialized.",
+      "1.3.5"),
   @Deprecated(since = "2.0.0")
   GENERAL_CLASSPATHS(org.apache.accumulo.start.classloader.AccumuloClassLoader.GENERAL_CLASSPATHS,
       "", PropertyType.STRING,
-      "This property is deprecated since 2.0.0. The class path should instead be configured"
+      "The class path should instead be configured"
           + " by the launch environment (for example, accumulo-env.sh). A list of all"
           + " of the places to look for a class. Order does matter, as it will look for"
           + " the jar starting in the first location to the last. Supports full regex"
-          + " on filename alone."),
+          + " on filename alone.",
+      "1.3.5"),
   @Deprecated(since = "1.7.0")
   @ReplacedBy(property = TABLE_DURABILITY)
   TSERV_WAL_SYNC_METHOD("tserver.wal.sync.method", "hsync", PropertyType.STRING,
-      "This property is deprecated since 1.7.0. Use table.durability instead."),
+      "Use table.durability instead.", "1.5.2"),
   @Deprecated(since = "1.7.0")
   @ReplacedBy(property = TABLE_DURABILITY)
   TABLE_WALOG_ENABLED("table.walog.enabled", "true", PropertyType.BOOLEAN,
-      "This setting is deprecated since 1.7.0. Use table.durability=none instead."),
+      "Use table.durability=none instead.", "1.3.5"),
   @Deprecated(since = "2.0.0")
   @ReplacedBy(property = TSERV_SCAN_EXECUTORS_DEFAULT_THREADS)
   TSERV_READ_AHEAD_MAXCONCURRENT("tserver.readahead.concurrent.max", "16", PropertyType.COUNT,
-      "This property is deprecated since 2.0.0, use tserver.scan.executors.default.threads "
-          + "instead. The maximum number of concurrent read ahead that will execute. This "
+      "The maximum number of concurrent read ahead that will execute. This "
           + "effectively limits the number of long running scans that can run concurrently "
-          + "per tserver.\""),
+          + "per tserver.\"",
+      "1.3.5"),
   @Deprecated(since = "2.0.0")
   @ReplacedBy(property = TSERV_SCAN_EXECUTORS_META_THREADS)
   TSERV_METADATA_READ_AHEAD_MAXCONCURRENT("tserver.metadata.readahead.concurrent.max", "8",
-      PropertyType.COUNT,
-      "This property is deprecated since 2.0.0, use tserver.scan.executors.meta.threads instead. "
-          + "The maximum number of concurrent metadata read ahead that will execute.");
+      PropertyType.COUNT, "The maximum number of concurrent metadata read ahead that will execute.",
+      "1.3.5");
 
   private final String key;
   private final String defaultValue;
   private final String description;
+  private final String deprecatedSince;
```

Comment:
I think this is causing the test failure
```suggestion
  private String deprecatedSince;
```